x11-misc/polybar/Manifest

```diff
@@ -1,3 +1,6 @@
 DIST i3ipcpp-0.7.1.tar.gz 191417 BLAKE2B 62b76a0921c78f374309d26856c01d1467b8bd181ce922fd47daec406afda9d76eea9a1f473b4899e22f99e2ef36fc70f72db1bb17eb1f3e71a8431c2634b915 SHA512 ef9f591bb4436916ad038bcb0c15ea3415d1978ff264fb276108ddac89c98515c464fbf252429f6a76589cb78e1434adba2efefb5a844dadad0e261f3806fb72
 DIST polybar-3.1.0.tar.gz 418298 BLAKE2B 1c7394a17ff1ec5b0dec519b7aa58e63cfcec0f104d4302051f6e12753b7074c3d85072a3d78e7ce135c53d4ddd4a6f68ea61ffb7b2ad04997bbf0c836115046 SHA512 6d8759d9f750c2abfa79fe735615b0ed3d1827e3a88561317712c6f02cb32a0a77726f97cb419f3a040ae9085a34a23cb1aad0aacf3f14e56c542977b89036b2
 DIST xpp-1.4.0.tar.gz 84095 BLAKE2B 2af4221953abe7f40473c55cd50c608b54ebc85a62be65af987ed5e6c5eaf56fbd92531788becf39ba5ad52099ac78f36e9b2a2515b473f4d86ed5fda3fa5065 SHA512 d34cbbe2391db36db7dffac0415d26b7487536e97c4d7efd472ab0aeffa4c8e757ee030b05e2aa1cfa8a591ee591bb96a6db7eb9fa5850027f958e92f078739c
+EBUILD polybar-3.1.0.ebuild 1670 BLAKE2B d7d09abb610cd6d0d8b506ecc1427770dae986deb64c2e6eb69dbe6dc80decfcf7ecd9c933c503f779195bc28fdd7c42991f53da6b6a4be5eff5718f075c4fa2 SHA512 ca8c8f39c8c8683a589bda2d49a54d2baad1d266e66968f4361dfd50bf1d5108587cefc0b0e13c00292a19a4799cf1803d80aa5c5e00ca24d0031444a1b390be
+EBUILD polybar-9999.ebuild 1177 BLAKE2B edd1dcc60f26bd303199f5a9b5a3e631cafd14c959731f36d34b68cbc96f6e19469cadbea788f179682d79f6f635bf22b16923985a16742d8a56b92d705ddb56 SHA512 1fc3f570215ca8569c46c431d83fd5dbc35f12a6ac2dcc8802de2981c54d24b1c83e53a9c4fbcac18fb7b66c3051e7ff1ad7ba2b6261c9fd69858c2483536636
+MISC metadata.xml 697 BLAKE2B d1ff2872e79a5037c85f9f5dbffcb73a24b7409d4c9b79fa2d81cf1af26048cb29ef3ca550b4293582f77f7da41286f0a39f2cd4957777a811b2368b5d9fb1c7 SHA512 5bd6c099b1eb2a4fe0bb83948c5980acd9725ae1a8d67bd7b2cdfb2adad37fe08df46fdc8fd551e64d1fd46ddaef6948816148e4fb2e1f627b555d88a98c78cb
```

Comment:
Looks like you have thick manifest layout configured in your clone/fork. Please check your metadata/layout.conf